.travis.yml

```diff
@@ -16,3 +17,5 @@ env:
   global:
   - SAUCE_USERNAME: Desire2Learn

```

Comment:
It'd be nice to get the comments back in here so we can tell which variable is which. I'm assuming that first one is `# SAUCE_ACCESS_KEY`.

Reply:
Agreed - added comments for those.  :)